v3.4 GLFW_ANY_POSITION on arm 32bit

I'm getting the following error:

```
# github.com/go-gl/glfw/v3.4/glfw
Error: ../../../go/pkg/mod/github.com/go-gl/glfw/v3.4/glfw@v0.1.0-pre.1/window.go:208:20: cannot use (_Ciconst_GLFW_ANY_POSITION) (untyped int constant 2147483648) as int value in constant declaration (overflows)
Error: Process completed with exit code 1.
```

The binding does:

```
AnyPosition int = C.GLFW_ANY_POSITION
```

And upstream defines:

```
#define GLFW_ANY_POSITION 0x80000000
```

So I guess it overflows the int size on 32bit?

See: https://github.com/libretro/ludo/actions/runs/24262354910/job/70849082030#step:20:283

--- a/v3.4/glfw/window.go
+++ b/v3.4/glfw/window.go
@@ -205,7 +205,7 @@ const (
 
 	// AnyPosition is a hint value for PositionX and PositionY that lets the
 	// window manager decide the initial window position.
-	AnyPosition int = C.GLFW_ANY_POSITION
+	AnyPosition int = -1 << 31 // C.GLFW_ANY_POSITION
 )
 
 // Window represents a window.

